### PR TITLE
move some packages to requirements-dev.txt [Resolves #83]

### LINKED ISSUE #83: Remove unnecessary dependencies in requirements.txt
For example:

https://github.com/dssg/tyra/blob/eedcbe1cdc210499518f6e8d4efae7005a9d8c29/requirements.txt#L8-L10

Also, are  `pandas` and `testing.postgresql` needed?

And these should be moved to `requirements-dev.txt`

https://github.com/dssg/tyra/blob/eedcbe1cdc210499518f6e8d4efae7005a9d8c29/requirements.txt#L13-L15

### PR DESCRIPTION
**Pull request by tweddielin**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-r requirements.txt
```
Comment (nanounanue) [inline requirements-dev.txt]: I think that this is not necessary, if the developer wants, he/she could install first the `requirements.txt` and then `requirements-dev.txt`

File: requirements.txt [hunk 2]
```diff
@@ -3,14 +3,10 @@ SQLAlchemy==1.0.14
 pandas==0.18.1
```
Comment (nanounanue) [inline requirements.txt]: I suggest to remove the version, and only add it when you deliver a new *stable* version

Comment (tweddielin) [reply requirements.txt]: okay I see. Do you suggest all the packages or just pandas?

**Comment by nanounanue** [comment]
:(

**Pull request merged**